src/renderers/shared/fiber/__tests__/ReactFiberHostContext-test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+var React;
+var ReactFiberReconciler;
+
+describe('ReactCoroutine', () => {
+  beforeEach(() => {
+    jest.resetModules();
+    React = require('React');
+    ReactFiberReconciler = require('ReactFiberReconciler');
+  });
+
+  it('works with null host context', () => {
```

Comment:
This is public-ish renderer API. I'd rather have a test just for this so we don't change it later accidentally.